lightning/src/ln/invoice_utils.rs

```diff
@@ -907,40 +900,29 @@ mod test {
 		// channel we'll immediately switch to including it as a route hint, even though it isn't
 		// yet announced.
 		let pub_channel_scid = mine_transaction(&nodes[0], &conf_tx);
-		let node_a_pub_channel_ready = get_event_msg!(
-			nodes[0],
-			MessageSendEvent::SendChannelReady,
-			nodes[1].node.get_our_node_id()
-		);
-		nodes[1]
-			.node
-			.handle_channel_ready(nodes[0].node.get_our_node_id(), &node_a_pub_channel_ready);
+		let node_a_pub_channel_ready =
+			get_event_msg!(nodes[0], MessageSendEvent::SendChannelReady, node_b_id);
+		nodes[1].node.handle_channel_ready(node_a_id, &node_a_pub_channel_ready);
 
 		assert_eq!(mine_transaction(&nodes[1], &conf_tx), pub_channel_scid);
 		let events = nodes[1].node.get_and_clear_pending_msg_events();
 		assert_eq!(events.len(), 2);
 		if let MessageSendEvent::SendChannelReady { msg, .. } = &events[0] {
-			nodes[0].node.handle_channel_ready(nodes[1].node.get_our_node_id(), msg);
+			nodes[0].node.handle_channel_ready(node_b_id, msg);
 		} else {
 			panic!();
 		}
 		if let MessageSendEvent::SendChannelUpdate { msg, .. } = &events[1] {
-			nodes[0].node.handle_channel_update(nodes[1].node.get_our_node_id(), msg);
+			nodes[0].node.handle_channel_update(node_b_id, msg);
 		} else {
 			panic!();
 		}
 
-		nodes[1].node.handle_channel_update(
-			nodes[0].node.get_our_node_id(),
-			&get_event_msg!(
-				nodes[0],
-				MessageSendEvent::SendChannelUpdate,
-				nodes[1].node.get_our_node_id()
-			),
-		);
+		let as_update = get_event_msg!(nodes[0], MessageSendEvent::SendChannelUpdate, node_b_id);
+		nodes[1].node.handle_channel_update(node_a_id, &as_update);
 
-		expect_channel_ready_event(&nodes[0], &nodes[1].node.get_our_node_id());
-		expect_channel_ready_event(&nodes[1], &nodes[0].node.get_our_node_id());
+		expect_channel_ready_event(&nodes[0], &node_b_id);
```

Comment:
Also after reviewing this PR, my opinion remains unchanged that changes like this aren't worth it. This PR is compact, so not a problem, but I'd be hesitant with future formatting changes that are anything more than what's the absolute undisputed minimum required for readability.

Reply:
If some people think the clean up makes it less readable (making it 'disputed'), I don't think we should do it and stick to the rustfmt output.